### ISSUE TITLE
Consider idempotent bower dist file naming

### ISSUE THREAD

**Issue by jhillyerd** [codepb/jquery-template#87]
I'm new to bower, but I've noticed that all my other modules don't include the version number on the files they place in dist, examples:
```
> ls jquery/dist/
jquery.js  jquery.min.js  jquery.min.map  jquery.slim.js  jquery.slim.min.js  jquery.slim.min.map
> ls jquery-sparkline/dist/
jquery.sparkline.js  jquery.sparkline.min.js
> ls bootstrap/dist/js/
bootstrap.js  bootstrap.min.js  npm.js
```

But jquery-load-template does:
```
> ls jquery-load-template/dist/
jquery.loadTemplate-1.5.6.js  jquery.loadTemplate-1.5.6.min.js
```

This means I have to edit the script tags in my HTML template(s) when updating.

**Comment by codepb** [comment]
Good point. I will look to add a separate folder containing the versioned files.

**Comment by codepb** [comment]
how should this be handled in source control, I'm not sure. Source control is obviously versioned anyway, and I tag each new release, is there any need for versioning in file names?

**Comment by jhillyerd** [comment]
I can't think of many projects that include version numbers in their own files.  However, I don't do that much JavaScript development, so I'm far from an expert!

**Comment by codepb** [comment]
Implemented in f9fb1a98f0ff2501776bb16f318ec39b97b1ab1f
